TestSolutions/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
```

Comment:
Lets add a comment here along these lines:

> This file is needed, with properties or without, to prevent importing any Directory.Build.props higher up in the directory tree.